assets/quill-cursors.scss

```diff
@@ -104,9 +104,6 @@ $transition-func: cubic-bezier(0.250, 0.460, 0.450, 0.940);
   }
 
   .ql-cursor-caret-container {
-    margin-left: -$spacing-sm;
-    padding: 0 $spacing-sm;
```

Comment:
This is here to make it easier to trigger the hover behaviour; I'm afraid we can't just delete it.